IPython/terminal/shortcuts/__init__.py

```diff
@@ -186,12 +186,12 @@ def create_identifier(handler: Callable):
     # 2) prompt-toolkit checks if we are at the end of text, not end of line
     #    hence it does not work in multi-line mode of navigable provider
     Binding(
-        auto_suggest.accept_in_vi_insert_mode,
+        auto_suggest.accept_or_jump_to_end,
```

Comment:
If a user configured shortcuts, their custom configuration will no longer work after this change. This is a twofold issue:
- if they overridden (modified/removed) this shortcut by matching by the old command name (`accept_in_vi_insert_mode`), the override will no longer work as it will not find a match (`accept_in_vi_insert_mode` is no longer here)
- if they added a new shortcut using the old command name, it will not work either as `accept_in_vi_insert_mode` command is not exposed anymore (currently we only allow commands which are in existing bindings; we could have a separate register of additional allowed commands which would alleviate this problem)

This is relatively minor breakage given that shortcuts were just introduced, and the name of this command was in a way misleading (it does not check for vi insert mode), so I don't have a strong opinion here.

Reply:
Good point, though if we don't emit a deprecation warning people will use the old name for ever. 

Let's try to break, and if we have complaints I can do a patch release.